indigo/distribution.yaml

```diff
@@ -3052,6 +3052,17 @@ repositories:
       url: https://github.com/WPI-RAIL/interactive_world.git
       version: develop
     status: maintained
+  iot_bridge:
+    doc:
+      type: git
+      url: dsd
```

Comment:
i used  the bloom package utility.  No idea where WPI-Rail, etc. came from, definitely not the entry I put into the bloom script.


I will check out rosdistro and issue a pull request. 


-----Original Message-----
From: Dirk Thomas <notifications@github.com>
To: ros/rosdistro <rosdistro@noreply.github.com>
Cc: corb555 <corb@aol.com>
Sent: Sun, Jun 21, 2015 5:23 pm
Subject: Re: [rosdistro] iot_bridge: 0.8.2-0 in 'indigo/distribution.yaml' [bloom] (#8763)


 
In indigo/distribution.yaml:
  
> @@ -3052,6 +3052,17 @@ repositories:
>        url: https://github.com/WPI-RAIL/interactive_world.git
>        version: develop
>      status: maintained
> +  iot_bridge:
> +    doc:
> +      type: git
> +      url: dsd

  
Invalid.
  
—
Reply to this email directly or view it on GitHub.